/**:
  ros__parameters:
    res: 0.05
    width: 80
    height: 80
    origin_x: 0.0
    origin_y: -2.0 #- (self.height * self.res) / 2.0
    dx: -0.7 #-0.34
    dy: 0.0
    closing_kernel_size: 1

